packages/plugin-commands-script-runners/src/runRecursive.ts

```diff
@@ -32,9 +32,11 @@ export default async (
     throw new PnpmError('SCRIPT_NAME_IS_REQUIRED', 'You must specify the script you want to run')
   }
   let hasCommand = 0
-  const packageChunks = opts.sort
+
+  const sortedPackageChunks = opts.sort
```

Comment:
I don't think reverse has sense if sorting is off. 

```suggestion
  const sortedPackageChunks = opts.sort || opts.reverse
```

Reply:
While its not necessary for my use-case, the packages are still consistently sorted when sort=false, they are just sorted lexically rather than topologically.

I went this way because it felt like more consistent and less surprising behaviour than reverse not doing anything when sort=false